arduino/netsender/NetSender.cpp

```diff
@@ -262,57 +309,67 @@ int setPins(const char * names, Pin * pins) {
   return sz;
 }
 
-// resetPowerPins resets all power pins.
-void resetPowerPins() {
+// resetPowerPins resets all power pins and writes the ESP32 alarm pin.
+// When alarm is true, all pins are set to LOW regardless of their default level.
+// For the ESP32, the separate alarm pin is also set.
+void resetPowerPins(bool alarm) {
+  int level;
   for (int ii = 0; ii < NUM_RELAYS; ii++) {
-    if (PowerPins[ii].alarm) {
-      continue;
+    level = PowerPins[ii].on ? HIGH : LOW;
+    if (alarm) {
+      level = LOW;
     }
     pinMode(PowerPins[ii].pin, OUTPUT);
-    digitalWrite(PowerPins[ii].pin, LOW);
-    if (Debug) Serial.print(F("Reset power pin: D")), Serial.println(PowerPins[ii].pin);
-  }
+    digitalWrite(PowerPins[ii].pin, level);
+    if (Debug) Serial.print(F("Set power pin: D")), Serial.print(PowerPins[ii].pin), Serial.print(F(" ")), Serial.println(fmtLevel(level));
```

Comment:
I think there is ```Serial.printf``` available. Also, could we have a ```debug``` function that checks ```Debug``` ?

Reply:
I didn't realize that ESP supported Serial.printf as the standard Arduino library does not . The origins of this code goes back to the Arduino Uno and predates the ESP8266 and ESP32!

I'll make a separate PR to tidy up debug logging.